bundles/net.tourbook/src/net/tourbook/messages_fr.properties

```diff
@@ -729,14 +736,14 @@ Dlg_AdjustAltitude_Radio_keep_start_altitude_tooltip  = Garder l'altitude du d\u
 Dlg_TourMarker_Button_HideAllMarker              = M&asquer tout
 Dlg_TourMarker_Button_HideAllMarker_Tooltip      = Masquer tous les marqueurs
 Dlg_TourMarker_Button_PasteFromClipboard_Tooltip = Ins\u00E9rer le texte depuis le presse-papier et le convertir afin de le rendre plus lisible.\n\
-                                                   \n\

```

Comment:
@wolfgang-ch  Here is an example of what happens when I just save my french translations using with the Messages Editor. Do you know why this happens ?